services/shotgrid_common/ayon_shotgrid_hub/__init__.py

```diff
@@ -306,6 +306,16 @@ def react_to_shotgrid_event(self, sg_event_meta):
                 f"Ignoring event, AYON project {self.project_name} not found.")
             return
 
+        # task revived only sends attribute_change event, before attribute
+        # change of parent asset AND finally entity_revival of Asset event
+        if (sg_event_meta["type"] == "attribute_change"
+            and sg_event_meta["attribute_name"] == "retirement_date"):
+            if sg_event_meta["entity_type"].lower() != "task":
+                # do nothing for update retirement_date on non existing Asset
+                return
+            self.log.info("changed to entity_revival")
```

Comment:
Can we make this log a bit clearer? This will be hard to grasp out of context if I'd see that log anywhere.